src/decimal128.ts

```diff
@@ -160,6 +160,7 @@ export class Decimal128 extends BSONValue {
   static fromString(representation: string): Decimal128 {
     // Parse state tracking
     let isNegative = false;
+    let sawSign = false;
```

Comment:
🎵 I saw the sign
And it opened up my eyes, I saw the sign
Life is demanding without understanding 🎵